tools/utils/seed/code_change_tools.ts

```diff
@@ -1,5 +1,6 @@
 import {BROWSER_SYNC_CONFIG} from '../../config';
 import * as browserSync from 'browser-sync';
+let path = require('path');
```

Comment:
When type definitions are available we're using ES2015 module imports:

```ts
import {parse} from 'path';
```

or

```ts
import * as path from 'path';
```